### PR TITLE
Replace another 2 .URL occurrences with .Permalink

### PR DESCRIPTION
**Pull request by strk**
This is basically a completion of #216

### CODE REVIEW

**Comment by strk** [comment]
\cc @budparr @jish @bstoney

File: layouts/partials/site-navigation.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <ul class="pl0 mr3">
           {{ range .Site.Menus.main }}
           <li class="list f5 f4-ns fw4 dib pr3">
-            <a class="hover-white no-underline white-90" href="{{ .URL }}" title="{{ .Name }} page">
+            <a class="hover-white no-underline white-90" href="{{ .Permalink }}" title="{{ .Name }} page">
```
Comment (budparr) [inline layouts/partials/site-navigation.html:12]: Did you test this? This one is incorrect. For menus, the `.URL` remains. https://gohugo.io/templates/menu-templates/

Comment (strk) [reply layouts/partials/site-navigation.html:12]: I did not test, other than verifying that the following annoying message goes away with the patch:
> WARN 2020/05/11 16:14:04 Page's .URL is deprecated and will be removed in a future release. Use .Permalink or .RelPermalink. If what you want is the front matter URL value, use .Params.url.

I'm new to Hugo so don't really know how to test site-navigation (and I'm actually missing it, see http://strk.kbt.io/blog/)

Comment (budparr) [reply layouts/partials/site-navigation.html:12]: No worries. We can just take this out. I'll have a look at the other one on my local, but should  be good.

**Comment by budparr** [comment]
The second one is fine. If you could remove the one in the navigation, please, I'll merge this.

**Comment by strk** [comment]
Done, and confirm the WARN does not re-surface

**Comment by budparr** [comment]
Thanks!

**Pull request merged**